marklogic-data-hub-central/ui/src/components/load/load-list.test.tsx

```diff
@@ -193,9 +193,9 @@ describe('Load data component', () => {
     //Check if the list is rendered properly
     expect(getByText('testLoadXML')).toBeInTheDocument();
 
-    fireEvent.mouseOver(getByLabelText('testLoadXML-add-icon')); // Hover over the Add to Flow Icon to get more options
+    fireEvent.click(getByLabelText('testLoadXML-add-icon')); // Hover over the Add to Flow Icon to get more options
```

Comment:
Action is a click event but the comment says 'Hover over...'.
Let's update the comments as well in such cases since they help to understand the related code better.